uplink/src/collector/tcpjson/util.rs

```diff
@@ -0,0 +1,52 @@
+use std::fmt::Display;
```

Comment:
We can extract this into a crate and publish. We can also raise a PR to tokio-util